pipeline/schemas/pipeline.py

```diff
@@ -56,6 +57,7 @@ class PipelineGet(PipelineGetBrief, RunnableGet):
     models: List[ModelGet]
     graph_nodes: List[PipelineGraphNode]
     outputs: List[str]
+    project: ProjectGet
```

Comment:
I think we can remove this for now so we don't have to deal with permissions for nested objects which may belong to different users.

Reply:
done